<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="stNum1">
       <property name="text">
        <string>ffffffff</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stNum2">
       <property name="text">
        <string>00000020</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stNum3">
       <property name="text">
        <string>00000026</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stNum4">
       <property name="text">
        <string>00000032</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>基站编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_begin">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="s1_x">
         <property name="text">
          <string>1856</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s2_x">
         <property name="text">
          <string>1885.6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s3_x">
         <property name="text">
          <string>523.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s4_x">
         <property name="text">
          <string>543</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="s1_y">
         <property name="text">
          <string>2141</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s2_y">
         <property name="text">
          <string>1416</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s3_y">
         <property name="text">
          <string>2141</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s4_y">
         <property name="text">
          <string>1427</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="s1_z">
         <property name="text">
          <string>230</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s2_z">
         <property name="text">
          <string>230</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s3_z">
         <property name="text">
          <string>230</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s4_z">
         <property name="text">
          <string>230</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_bound">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>绑定</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="group_Num">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="group_Tag_Num">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ip">
       <property name="text">
        <string>192.168.1.202</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_port">
       <property name="text">
        <string>60000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>81</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>基站数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>分组数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>每组标签数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
